.github/CODEOWNERS

```
@@ -1,2 +1,2 @@
-* @im-open/swat @im-open/infra-purple
+* @im-open/swat @im-open/infra-purple @jsclifford
```

Instead of adding yourself to the list, you can just be the list.  We put our template in there like that so we can do the initial review and make sure the actions are conforming to the standards, but after that you will own the implementation and support of this action.
```suggestion
* @jsclifford
```

---

I committed your suggestion.